tests/test_cram.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+import os
+import sys
+import warnings
+import unittest
+import subprocess
+import logging
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from xsamtools import cram  # noqa
+
+WORKSPACE_BUCKET = 'fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10'
+log = logging.getLogger(__name__)
+
+
+class TestCram(unittest.TestCase):
+    def setUp(self):
+        # Suppress the annoying google gcloud _CLOUD_SDK_CREDENTIALS_WARNING warnings
+        warnings.filterwarnings('ignore', 'Your application has authenticated using end user credentials')
+        # Suppress unclosed socket warnings
+        warnings.simplefilter('ignore', ResourceWarning)
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.cram_gs_path = 'gs://lons-test/ce#5b.cram'
```

Comment:
Will this fixture always be available?

If it's small enough it can be checked it into the repo and populated it into a bucket for testing.

Reply:
Hmmm... I suppose if the fixture was ever deleted from the bucket by accident, that could be a problem.

I'll add it to the fixtures, but I almost feel like our fixtures should be in and fetched from a separate repo.  And then maybe if and only if they are not found in the buckets.  I think that that would help keep clones of xsamtools slim, especially since I predict a well-developed xsamtools should inevitably accumulate a lot of test fixtures for crams, bams, vcfs, etc.

I have experienced hitting a 1Gb free upload limit per month before when cloning a repo, and since then have tried to keep the size down.  Though that was explicit before and it appears as though github may have changed their policy to be a bit more nebulous?  https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/what-is-my-disk-quota